internal/tracinginterceptor/interceptor.go

```diff
@@ -177,12 +177,53 @@ func (i *Interceptor) CallOneway(ctx context.Context, req *transport.Request, ou
 
 // HandleStream implements interceptor.StreamInbound
 func (i *Interceptor) HandleStream(s *transport.ServerStream, h transport.StreamHandler) error {
-	panic("implement me")
+	parentSpanCtx, _ := i.tracer.Extract(i.propagationFormat, getPropagationCarrier(s.Request().Meta.Headers.Items(), i.transport))
+
+	// Start the server-side span
+	span := i.tracer.StartSpan(
+		s.Request().Meta.Procedure,
+		opentracing.ChildOf(parentSpanCtx),
+		ext.SpanKindRPCServer,
+		opentracing.StartTime(time.Now()),
+	)
+	defer span.Finish()
+
+	// Wrap the ServerStream in a tracedServerStream
+	err := h.HandleStream(s)
+	tracedStream := newTracedServerStream(*s, span, i.log)
+
+	return updateSpanWithErrorDetails(span, tracedStream.IsApplicationError(), tracedStream.ApplicationErrorMeta(), err)
```

Comment:
tracedStream just created, what and where is suppose to set IsApplicationError and ApplicationErrorMeta values?